### ISSUE TITLE
Listbox option aria-label not read during browse mode

### ISSUE THREAD

**Issue by akhileshmuthusamy-rideco** [nvaccess/nvda#20259]
### Brief summary

In browse mode, NVDA does not read the `aria-label` assigned to the listbox option.

### Steps to reproduce

In Chrome, go to this example https://jsfiddle.net/5hzyav8L/2/ and set the focus on the `Next` button and click `Ctrl + down arrow`.

##### Speech Viewer Output:
```
Next button
Supported days list   4
button End
```
In Chrome, go to this example https://jsfiddle.net/5hzyav8L/3/ and set the focus on the `Next` button and click `Ctrl + down arrow`.

##### Speech Viewer Output:
```
Next button
Supported days list
button End
```

### Actual behavior

In this https://jsfiddle.net/5hzyav8L/3/ example, NVDA does not read the aria label "Day 4", whereas in this https://jsfiddle.net/5hzyav8L/2/ example, it reads the first option content "4".

### Expected behavior

NVDA reads the aria-label in the listbox option in browse mode.

### NVDA type

installed copy

### NVDA version

2025.3.3

### Windows version

Windows 11 Pro Version 25H2 Build 26200.8457 

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by seanbudd** [comment]
Can you please also report to Chromium?

**Comment by akhileshmuthusamy-rideco** [comment]
@seanbudd I have reported the issue to Chromium https://issues.chromium.org/issues/519218480 .

I am also seeing a similar issue with Firefox.

### Steps to reproduce
In Firefox, go to this example https://jsfiddle.net/5hzyav8L/2/ and set the focus on the Next button and click Ctrl + down arrow.

Speech Viewer Output:
```
Next button
Supported days list   4
button End
```
In Firefox, go to this example https://jsfiddle.net/5hzyav8L/3/ and set the focus on the Next button and click Ctrl + down arrow.

Speech Viewer Output:
```
Next button
Supported days list  4
button End
```
### Actual behavior
In this https://jsfiddle.net/5hzyav8L/3/ example, NVDA reads the option content "4" instead of reading the aria label "Day 4", whereas in this https://jsfiddle.net/5hzyav8L/2/ example, it reads the first option content "4".

### Expected behavior
NVDA reads the aria-label in the listbox option in browse mode.
```
Next button
Supported days list  Day 4
button End
```

**Comment by seanbudd** [comment]
Yes it seems like we are not receiving the information in the virtual buffer. Could you also report to mozilla? thanks

**Comment by akhileshmuthusamy-rideco** [comment]
Thanks for checking. Reported the issue to Mozilla https://bugzilla.mozilla.org/show_bug.cgi?id=2052549

**Comment by akhileshmuthusamy-rideco** [comment]
Firefox team says the issue is with NVDA (https://bugzilla.mozilla.org/show_bug.cgi?id=2052549). Could you please check the issue again?

> Firefox is exposing everything correctly here, including both the accName and the explicit-name:true attribute (which indicates that the name was explicitly specified by the author). This needs to be addressed in NVDA.
